TheMovieDb/Controllers/ListViewController.swift

```diff
@@ -82,7 +107,7 @@ class ListViewController: UICollectionViewController {
                         self.collectionView.reloadData()
                     }
                 case .failure(let error):
-                    print("The error \(error.localizedDescription)")
+                    print("The error fetchData \(error.localizedDescription)")
```

Comment:
Maybe in future iterations of the app we could change this part to manage the error more appropriately, like showing an alert or a error view, or something like that.

Reply:
Yes, I will provide a better solution